src/store/index.js

```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+const store = new Vuex.Store({
+    state: {
+        count: 10
```

Comment:
please add a comment stating the significance of 10

Reply:
ok